essencium-backend/src/main/java/de/frachtwerk/essencium/backend/controller/AbstractAccessAwareController.java

```diff
@@ -209,6 +209,7 @@ public List<BasicRepresentation> findAll(@Parameter(hidden = true) SPEC specific
       required = true,
       content = @Content(schema = @Schema(type = "integer")))
   public REPRESENTATION findById(
+      @PathVariable("id") @NotNull final ID id,
```

Comment:
Maybe we were talking past each other: we don't need the ID as a parameter here, so it can be removed. We're willing to accept the "unbind"-hint from the IDE/Sonar.

Reply:
Id parameter is removed again.